doc/zinit.1

```diff
@@ -138,15 +138,15 @@ A new ice-mod \fBreset''\fR that ivokes \fBgit reset --hard\fR (or the provided
 Three new zinit annexes (i.e. \fBextensions\fR \fI\(lahttps://zdharma-continuum.github.io/zinit/wiki/Annexes/\(ra\fR):
 .RS 4
 .IP \(bu 4
-\fBz-a-man\fR \fI\(lahttps://github.com/zinit/z-a-man\(ra\fR
+\fBzinit-annex-man\fR \fI\(lahttps://github.com/zinit/zinit-annex-man\(ra\fR
 .P
 Generates man pages and code-documentation man pages from plugin's README.md and source files (the code documentation is obtained from \fBZshelldoc\fR \fI\(lahttps://github.com/zdharma/zshelldoc\(ra\fR).
```

Comment:
missed

Reply:
i could be wrong, reviewing on phone

Reply:
Looks good to me, groff is just weird to look at, even on a computer monitor.

https://www.gnu.org/software/groff/manual/groff.html